ReactAndroid/src/main/java/com/facebook/react/views/webview/ReactWebViewManager.java

```diff
@@ -102,6 +96,11 @@ public void onPageStarted(WebView webView, String url, Bitmap favicon) {
       mLastLoadFailed = false;
 
       ReactContext reactContext = (ReactContext) ((ReactWebView) webView).getContext();
+
+      if (ReactBuildConfig.DEBUG && Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
```

Comment:
Can you just move this to createView? It's guaranteed to execute on the main thread